src/main/scripts/includes/install_archives

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+_archives_check_prereqs() {
+  if ! which bsdtar >/dev/null; then
+    echo "No bsdtar -> apt install libarchive-tools"
+    exit 1
+  fi
+  if ! gh auth status >/dev/null 2>&1; then
+    gh auth login -h github.com
+  fi
+}
+
+# shellcheck disable=SC2016
+_zips_add_path() {
+  path_to_add=$1
+  if [[ -z "$DPM_SKIP_ARCHIVES_PROFILE" ]]; then
+    if ! grep "$path_to_add" "$HOME/.bashrc" >/dev/null 2>&1; then
```

Comment:
Should `"$HOME/.bashrc"` be a variable, so it could be set `export DPM_PROFILE_FILE="$HOME/.bash_profile`. (or something better than `DPM_PROFILE_FILE`)

will raise an issue as its very much out of scope for this.

Reply:
#268